designs/virtualization/4.x/cd roms/cd roms.md

```diff
@@ -0,0 +1,39 @@
+---
+parent: Virtualization
+version: 4.x
+---
+# CD ROMs
+
+This PR deals with removing CD-ROMs from the details view and including them in the disks tab.
+
+## CD-ROMs on VM Details page
+
+We can remove CD ROMs from the VM Details view
```

Comment:
```suggestion
It's been determined that separating CD-ROMs from the rest of the disks has technical overhead and complicates the flow. It makes more sense to include them as a lower level item in the disks rather than highlight them in the details view like we currently do.
```

Reply:
thanks @matthewcarleton